pkg/services/contexthandler/contexthandler.go

```diff
@@ -128,6 +129,8 @@ func (h *ContextHandler) Middleware(next http.Handler) http.Handler {
 			reqContext.IsSignedIn = !reqContext.SignedInUser.IsAnonymous
 			reqContext.AllowAnonymous = reqContext.SignedInUser.IsAnonymous
 			reqContext.IsRenderCall = identity.AuthenticatedBy == login.RenderModule
+			ctx = appcontext.WithUser(ctx, reqContext.SignedInUser)
```

Comment:
@ryantxu seems like this one is only used for certain things like API server and entity server currently. Anything stopping us from making it available for any service in Grafana? 

Have there been any discussion switching from SignedInUser to this new [identity.Requester interface](https://github.com/grafana/grafana/blob/main/pkg/services/auth/identity/requester.go)? Any discussion making `identity.Requester` propagate through context? Maybe @kalleep or your teammates can provide some additional context?

Reply:
Yes the idea is to eventually replace `SignedInUser` with this interface. I know that @Jguer did a lot of work on this for a while. But for us to pass the interface here instead we would need to replace the usage of `SignedInUser` in all inner services before

Reply:
Make sense thanks

Reply:
regardless of path -- we will depend on always running in a context that lets you get the user details from ctx.  (this is how k8s works)

Reply:
I get that, but if we start using this in existing/new Grafana service either used in Grafana backend, an API server or both we need to make sure the context is populated correctly.